_includes/propdefs/head.md

```diff
@@ -34,7 +34,7 @@ definitions.
 
 ###### curies
 
-optional. List of
+optional. Map of
```

Comment:
I believe this should say "Array of". JSON RFC doesn't define anything called a "map", it only defines: object, array, string and number https://tools.ietf.org/html/rfc7159#page-3

Reply:
The change I was trying to make was to JSON object, not to JSON array.  The reason I wanted to change the `curies` value to an object/map was that the array version makes it easy to accidentally collide prefixes, where the object form would trivially prevent it.

Reply:
Yeah, I edited my comment.